docs/common/cursor_c_lua.ldoc

```diff
@@ -0,0 +1,80 @@
+-- <div class='flex-list'>
```

Comment:
This file should be named `cursor.ldoc`. the `_c` is to indicate the comments syntax is C style instead of Lua.

Reply:
Fair point. Let's just ignore my comment then because making these filenames more consistent is out of the scope of this PR 🤷

Reply:
Ah you did the change for this file anyway. Nice 👍

Reply:
> Ah you did the change for this file anyway. Nice 👍

Thanks. I might open another PR for changing those file names.